static/scss/layout.scss

```diff
@@ -87,9 +94,15 @@ $modal-sidebar-width: 540px;
   padding: 15px 15px 80px;
 }
 
-.modal.right.fade .modal-dialog {
-  right: -$modal-sidebar-width;
-  transition: opacity 0.3s linear, right 0.3s ease-out;
+.modal.right.fade {
```

Comment:
Great idea nesting the `.modal-dialog` styles above. Could we do the same for the `.modal-content` styles at line 88 and these `.modal.right.fade` styles here?

Reply:
I don't see why not!